Jenkinsfile

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env groovy
 common {
   slackChannel = '#rest-proxy-eng'
-  upstreamProjects = 'confluentinc/schema-registry'
+  downStreamRepos = ["confluent-security-plugins", "ce-kafka-rest",
```

Comment:
any thoughts on why we replaced upstream with downstream here?

Reply:
@mohnishbasha We are moving to a downstream model instead of upstream because it enables us to automate this in the future by using the dependency graph. Using the dependency graph we will no longer need to define these dependencies as we can just determine it from the graph. Having these build dependencies correct becomes more critical with nano versioning so we don't want to rely on a manual process to get it right and keep it updated.